files/setup-09-ingress.sh

```diff
@@ -137,6 +137,11 @@ spec:
     services:
     - name: vmware-event-router
       port: 8082
+  - conditions:
```

Comment:
This ingress setup works without issues right?

Reply:
That's correct